website/src/util/getSequencesByLocationMapData.ts

```diff
@@ -0,0 +1,93 @@
+import type { MapSource } from '@genspectrum/dashboard-components/util';
+import type { Topology } from 'topojson-specification';
+
+let knownMapFiles: ReturnType<typeof loadKnownMapFiles> | null = null;
+
+/* eslint-disable @typescript-eslint/naming-convention -- the keys must match the country names */
+const mapMetadata: Record<string, Omit<MapData, 'mapSource'>> = {
+    Germany: {
+        zoom: 5.8,
+        offsetX: 10,
+        offsetY: 51.4,
+    },
+    Switzerland: {
+        zoom: 7.7,
+        offsetX: 8.2,
+        offsetY: 46.8,
+    },
+    USA: {
+        zoom: 4.1,
```

Comment:
Just to be sure: we don't want to have a zoom level, where Alaska and Hawaii are shown? Or is there also another map available?

Reply:
We agreed that for now it's good enough, especially because users can scroll themselves. We can tweak the map later if we think that it's necessary.